stable/traefik/templates/configmap.yaml

```diff
@@ -88,17 +88,32 @@ data:
       {{- end}}
       {{- end}}
     {{- end }}
-    {{ if .Values.metrics.prometheus.enabled }}
+    {{- if .Values.metrics.prometheus.enabled }}
     [web.metrics.prometheus]
-      Buckets = [{{ .Values.metrics.prometheus.buckets }}]
+      {{- if .Values.metrics.prometheus.buckets }}
+      buckets = [
+        {{- range $idx, $element := .Values.metrics.prometheus.buckets }}
+            {{- if $idx }},{{ end }}
+            {{- printf "%.1f" $element }}
+        {{- end -}}
+        ]
+        {{- end -}}
     {{- end }}
-    {{ if .Values.metrics.datadog.enabled }}
+    {{- if .Values.metrics.datadog.enabled }}
     [web.metrics.datadog]
-      Address = {{ .Values.metrics.datadog.address }}
-      PushInterval = "{{ .Values.metrics.datadog.pushInterval }}"
+      {{- if .Values.metrics.datadog.address }}
+      address = {{ .Values.metrics.datadog.address | quote }}
+      {{- end}}
+      {{- if .Values.metrics.datadog.pushInterval }}
+      pushInterval = {{ .Values.metrics.datadog.pushInterval | quote }}
```

Comment:
Just checked the docs. `pushinterval` is all lower case. 

https://docs.traefik.io/configuration/backends/web/#statsd

It looks like this is pretty inconsistent. Some start with a lower case letter, some with an upper case letter, some are camel case, some all lower case. Oh my...

Reply:
pushinterval is now lowercase, in line with all the other variables.

The docs are inconsistent, Traffic doesn't seem seem to care about the case of the config variables...yet